### ISSUE TITLE
I need to reproduce an Azure media file but it seems I can not get it to work

### ISSUE THREAD

**Issue by milodude** [flutter/flutter#91090]
## Use case

I'm trying to reproduce an Azure Media File like this 'https://myURL.ism/manifest' but I have not been able to reproduce the live streaming.

`@override`
`void initState() {`
`super.initState();`
` videoPlayerController = VideoPlayerController.network(`
`'https://myURL.ism/manifest(format=m3u8-cmaf)', formatHint: VideoFormat.hls);`
` videoPlayerFuture = videoPlayerController.initialize();`
`    }`
## Question

Is there any configuration I am missing for this to work properly? I have seen this in stackoverlfow but it did not help much.

[stackAnswer](https://stackoverflow.com/questions/58844066/azure-media-services-flutter)

**Comment by TahaTesser** [comment]
Hi @milodude
`https://myURL.ism/manifest(format=mpd-time-csf)', formatHint: VideoFormat.dash);`
Are you ending the video with proper extension?  Looks like no extension here

If the problem persists, can you please provide your `flutter doctor -v` with the upgrade, complete reproducible minimal code sample, and complete `flutter run --verbose` logs running the minimal code sample?
Thank you

**Comment by milodude** [comment]
Hi taha, 

Firs of all thank you for your quick response. 
1- 
![image](https://user-images.githubusercontent.com/4110660/135861674-90e2fad9-a170-4825-b7bb-10ac11687561.png)

2-`import 'package:flutter/material.dart';
import 'package:video_player/video_player.dart';

class TestVideo extends StatefulWidget {
  static const String routeName = '/video';
  @override
  _TestVideoState createState() => _TestVideoState();
}

class _TestVideoState extends State<TestVideo> {
  VideoPlayerController videoPlayerController;
  Future<void> videoPlayerFuture;

  @override
  void initState() {
    super.initState();
    videoPlayerController = VideoPlayerController.network(
        'yourVideoUrl.ism/manifest(format=mpd-time-csf)',
        formatHint: VideoFormat.dash);
    videoPlayerFuture = videoPlayerController.initialize();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        appBar: AppBar(title: Text('Flutter Video Player Sample')),
        body: FutureBuilder(
          future: videoPlayerFuture,
          builder: (context, snapshot) {
            if (snapshot.connectionState == ConnectionState.done) {
              return AspectRatio(
                aspectRatio: videoPlayerController.value.aspectRatio,
                child: VideoPlayer(videoPlayerController),
              );
            } else {
              return Center(child: CircularProgressIndicator());
            }
          },
        ),
        floatingActionButton: FloatingActionButton(
          onPressed: () {
            setState(() {
              videoPlayerController.value.isPlaying
                  ? videoPlayerController.pause()
                  : videoPlayerController.play();
            });
          },
          child: Icon(
            videoPlayerController.value.isPlaying
                ? Icons.pause
                : Icons.play_arrow,
          ),
        ));
  }

  @override
  void dispose() {
    videoPlayerController.dispose();
    super.dispose();
  }
}`

This example was taken from the original package. If you use any mp4 video it works properly.

3-
![image](https://user-images.githubusercontent.com/4110660/135863068-d8051ae9-fd45-4e5e-8e91-386bd9692e98.png)

![CaptureBug1](https://user-images.githubusercontent.com/4110660/135863021-155d7735-8564-4372-a4f4-c73efb426544.PNG)


I have been using formatHint to complete the extension for a dash file. Also tried adding .mpd in the end of the url to test it manually.

Thanks in advance.

**Comment by TahaTesser** [comment]
Hi @milodude 
Thanks for all the details, Unfortunately, I don't have a test link for Azure Media File to reproduce

**Comment by TahaTesser** [comment]
@milodude 
Is the issue only on the web? Does it stream when running on mobile?

**Comment by milodude** [comment]
Yes, it works properly in mobile. Web it is the thing. Dunno if theres any solution about this.

![image](https://user-images.githubusercontent.com/4110660/136042986-c11c3107-5401-46c8-aed6-375a96258b2c.png)

**Comment by stuartmorgan** [comment]
Are you able to play that video in the browser in a non-Flutter context? (E.g., a simple page with `<video>`).

**Comment by milodude** [comment]
Yes, I tried it in https://ampdemo.azureedge.net/ where it supports live streaming.

**Comment by milodude** [comment]
> > Hello, I'm sorry to be persistent but was wondering if there is any updates on this issue.
> 
> Nobody is actively working on this issue, so there are no updates. If there were updates, you would see them here, so it's not necessary to ask.
> 
> > Let me know if I can help you somehow.
> 
> Anyone interested in investigating this further is certainly welcome to do so.

Hello, I have talked to my PM and he has agreed for me to search for a way  to fix this issue. They are willing to pay for your work in order to push these changes in. Would you be interested in make these changes? If so I would appreciate if you contact me and send me a budget estimation. Thanks in advanced.
